bundles/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/handler/Atd100Handler.java

```diff
@@ -0,0 +1,117 @@
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.satel.internal.handler;
+
+import static org.openhab.binding.satel.internal.SatelBindingConstants.*;
+
+import java.util.Collections;
+import java.util.Set;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.openhab.binding.satel.internal.command.ReadZoneTemperature;
+import org.openhab.binding.satel.internal.command.SatelCommand;
+import org.openhab.binding.satel.internal.config.Atd100Config;
+import org.openhab.binding.satel.internal.event.SatelEvent;
+import org.openhab.binding.satel.internal.event.ZoneTemperatureEvent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link Atd100Handler} is responsible for handling commands, which are
+ * sent to one of the channels of a ATD-100 device.
+ *
+ * @author Krzysztof Goworek - Initial contribution
+ */
+public class Atd100Handler extends WirelessChannelsHandler {
+
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = Collections.singleton(THING_TYPE_ATD100);
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    private ScheduledFuture<?> pollingJob;
+
+    public Atd100Handler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void initialize() {
+        super.initialize();
+
+        if (pollingJob == null || pollingJob.isCancelled()) {
+            Atd100Config config = getConfigAs(Atd100Config.class);
+            Runnable pollingCommand = () -> {
+                if (bridgeHandler.getThing().getStatus() == ThingStatus.ONLINE) {
+                    bridgeHandler.sendCommand(new ReadZoneTemperature(thingConfig.getId()), true);
+                }
+            };
+            pollingJob = scheduler.scheduleWithFixedDelay(pollingCommand, 0, config.getRefresh(), TimeUnit.MINUTES);
+        }
+    }
+
+    @Override
+    public void dispose() {
+        logger.debug("Disposing thing handler.");
+
+        if (pollingJob != null && !pollingJob.isCancelled()) {
+            pollingJob.cancel(true);
+            pollingJob = null;
+        }
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        if (CHANNEL_TEMPERATURE.equals(channelUID.getId())) {
+            logger.debug("New command for {}: {}", channelUID, command.toFullString());
```

Comment:
Simply pass `command`. This will call `toString`. Command internally call `toFullString`. But when simply passing command the `toString` method is not called when log level is not set to debug. With `toFullString`, it is always evaluated even when log level is not set to debug.
```suggestion
            logger.debug("New command for {}: {}", channelUID, command);
```

Reply:
You are right, done.